app/controllers/solidus_square/api/checkout_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 module SolidusSquare
   module Api
-    class CheckoutController < BaseController
+    class CheckoutController < ::SolidusSquare::BaseController
```

Comment:
I'm curios,
BaseController is under SolidusSquare namespace, are you sure that you need to specify the parent namespace? 🤔

Reply:
https://github.com/nebulab/solidus_square/pull/49/commits/591b5191d9fd9f444e56482bf3019ac8c5393bfb